plugins/elasticsearch/elasticsearch.go

```diff
@@ -166,43 +212,66 @@ func (o *ElasticSearchOutput) receiver(or OutputRunner, wg *sync.WaitGroup) {
 				outBatch = append(outBatch, outBytes...)
 				if count = count + 1; o.bulkIndexer.CheckFlush(count, len(outBatch)) {
```

Comment:
I'm only ever writing to count from the Run goroutine, despite the fact that committer reads the pointed-to int. Does it still need to be atomic? I've got no problem adding that, I just didn't think it needed to be atomic with 1 writer and multiple readers.